central/role/datastore/defaults.go

```diff
@@ -0,0 +1,220 @@
+package datastore
+
+import (
+	"github.com/pkg/errors"
+	rolePkg "github.com/stackrox/rox/central/role"
+	"github.com/stackrox/rox/central/role/resources"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/auth/permissions"
+	permissionsUtils "github.com/stackrox/rox/pkg/auth/permissions/utils"
+	"github.com/stackrox/rox/pkg/defaults/accesscontrol"
+	"github.com/stackrox/rox/pkg/env"
+	"github.com/stackrox/rox/pkg/utils"
+)
+
+type permSetAttributes struct {
+	idSuffix           string
+	postgresID         string // postgresID should be populated with valid UUID values.
```

Comment:
feel free to ignore, but since we are anyways touching this: might be worthwhile to switch to `uuid.UUID` type here instead of leaving it as string, this way we don't get non-UUID values by mistake.

Reply:
I will leave the intimate changes to you team